pkg/controller/logstorage/logstorage_controller_test.go

```diff
@@ -769,6 +769,56 @@ var _ = Describe("LogStorage controller", func() {
 					mockStatus.AssertExpectations(GinkgoT())
 				})
 
+				It("should not add OwnerReference to user supplied kibana TLS cert", func() {
```

Comment:
Could you add a test that the KibanaCertSecret is created with an OwnerReference when one is not provided? I don't see a test for that. We would have previously had that in the logstorage render.

Reply:
Created a new test for this case